cli/kata-check_ppc64le.go

```diff
@@ -37,11 +44,6 @@ func archHostCanCreateVMContainer() error {
 // hostIsVMContainerCapable checks to see if the host is theoretically capable
 // of creating a VM container.
 func hostIsVMContainerCapable(details vmContainerCapableDetails) error {
-	_, err := getCPUInfo(details.cpuInfoFile)
-	if err != nil {
-		return err
-	}
-
 	count, err := checkKernelModules(details.requiredKernelModules, archKernelParamHandler)
```

Comment:
I have a concern about modules check. What will happen here, if we are running with a built-in kernel : ) ?

Reply:
That's handled :)